app/src/util/Wallets2Tokens.js

```diff
@@ -215,8 +216,8 @@ class Wallets2Tokens {
 
     // this.$store.state.wallets.portfolioTotal = this.$store.state.wallets.portfolioTotal + usdValue * t.amount
     // console.log('this.$store.state.wallets', this.$store.state)
-    store.commit('wallets/updateTokens', this.tableData)
-    store.commit('wallets/updatePortfolioTotal', store.state.wallets.portfolioTotal)
+    // store.commit('wallets/updateTokens', this.tableData)
+    // store.commit('wallets/updatePortfolioTotal', store.state.wallets.portfolioTotal)
   }
   async getAllAssets () {
     await axios.get('https://volentix.info/get_assets')
```

Comment:
Interesting....

What is that?  Where is the repo for this?
Where is this server hosted?

Reply:
Are you referring to getAllAssets() ? It found it there.

I am updating the wallet elsewhere...